server/controllers/index.js

```diff
@@ -151,8 +156,30 @@ export const updateKatathon = async (req, res) => {
 }
 
 
-export const nextKatathon = (req, res) => {
-  res.send('Next Katathon is coming soonddd')
+export const nextKatathon = async (req, res) => {
+  try {
+    const katathons = await Katathon.find({ completed: false }).sort({ date_created: -1 })
+    if(katathons.length > 0) {
+      const nextKatathon = getNextEvent(katathons)
+      res.status(200).json({
+        result: 'Success',
+        data: nextKatathon,
+        message: 'Next Katathon found successfully'
+      })
+    }else {
+      res.status(404).json({
+        result: '404',
+        data: [],
+        message: 'No Katathon event found'
+      })
+    }
+  } catch (err) {
+    res.status(400).json({
+      result: 'failed',
```

Comment:
Maybe overly picky, but capitalise `Failed` for consistency with `Success` and `Not Found`, suggested above.